PathWeaver/Paths/3mTest.path

```diff
@@ -0,0 +1,3 @@
+X,Y,Tangent X,Tangent Y,Fixed Theta,Reversed,Name
+0.0,-8.2296,1.0,0.0,true,false,

```

Comment:
Why is y -8.2296

Reply:
Was just an artifact of where I visually created it on the field. It will be changed to 0.0 in the next commit